src/window.js

```diff
@@ -205,6 +210,19 @@ class Window extends Adw.ApplicationWindow {
     this._content_header_bar.pack_start(this._box_navigation);
   };
 
+  #createURLBar() {
+    this._url_bar = new URLBar();
```

Comment:
I was trying to take advantage of the fact that the headerbar automatically takes the title from content page when ``title-widget`` is set to null. Otherwise ig I'd have to switch between the URLBar and a ``Adw.WindowTitle`` on a stack which might help with removing all the boilerplate code for URLBar